tests/phpunit/integration/Core/Remote_Features/Remote_Features_SyncerTest.php

```diff
@@ -34,12 +34,16 @@ public function test_pull_remote_features() {
 			$this->setting,
 			fn () => array( 'testFeature' => array( 'enabled' => true ) )
 		);
+
 		$this->assertEquals( array(), $this->setting->get() );
 
 		$syncer->pull_remote_features();
 
 		$this->assertEquals(
-			array( 'testFeature' => array( 'enabled' => true ) ),
+			array(
+				'testFeature'     => array( 'enabled' => true ),
+				'last_updated_at' => time(),
```

Comment:
This is guaranteed to fail randomly in the future. Please refer to how other tests are done that involve the use of `time()` to avoid this fragility.